test/core/BehaviorTree-Serialization.js

```diff
@@ -149,7 +148,7 @@ suite('Core: Behavior Tree - Serialization', function() {
 
     test('Dump JSON model', function() {
         var tree = new BehaviorTree();
-        var CustomNode = Class(Condition);
+        var CustomNode = Condition;
```

Comment:
same as above. Perhaps we can remove this line and replace all the occurrences of `CustomNode` with `Condition`